webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -338,6 +338,7 @@ <h2>{{ report.product }} {{ report.version }} Crash Report [@ {{ report.signatur
                   <td>{{ report.address }}</td>
                 </tr>
                 {% if request.user.has_perm('crashstats.view_pii') or your_crash %}
+                  {# FIXME(willkg): switch this to java_stack_trace_raw after January 2020 #}
```

Comment:
We have to show the old one here because that's the one that has data for the last 6 months. When we delete the old one, we can switch to the new one here.